src/home/components/components/pokemon-detail/styles/pokemon-detail-styles.ts

```diff
@@ -0,0 +1,110 @@
+import styled from 'styled-components'
+
+// This component should stay into same shared folder
+export const CenterDiv = styled.div`
```

Comment:
Looks like an old and not very clean trick here with top, left and transform props ...